src/subscription_manager/managercli.py

```diff
@@ -744,6 +744,10 @@ def _do_command(self):
 
         try:
             self._validate_options()
```

Comment:
I'm sorry. I forgot to check behavior in all cases after implementing your previous suggestion. :-(